diff --git a/skmob/core/tests/test_flowdataframe.py b/skmob/core/tests/test_flowdataframe.py
index 631803d..4bbaee7 100644
--- a/skmob/core/tests/test_flowdataframe.py
+++ b/skmob/core/tests/test_flowdataframe.py
@@ -1,8 +1,8 @@
 import pytest
 import numpy as np
-from ...utils import constants
+from skmob.utils import constants
 import geopandas as gpd
-from ...core.flowdataframe import FlowDataFrame
+from skmob.core.flowdataframe import FlowDataFrame
 import folium
 
 
diff --git a/skmob/core/tests/test_trajectorydataframe.py b/skmob/core/tests/test_trajectorydataframe.py
index 6ed1a5f..0cbd92b 100644
--- a/skmob/core/tests/test_trajectorydataframe.py
+++ b/skmob/core/tests/test_trajectorydataframe.py
@@ -3,11 +3,11 @@ import pandas as pd
 import geopandas as gpd
 from datetime import datetime
 from operator import itemgetter
-from ...utils import constants
-from ...utils.constants import UID, DATETIME, LATITUDE, LONGITUDE, GEOLIFE_SAMPLE
-from ...core.trajectorydataframe import TrajDataFrame
-from ...core.flowdataframe import FlowDataFrame
-from ...preprocessing import detection, clustering
+from skmob.utils import constants
+from skmob.utils.constants import UID, DATETIME, LATITUDE, LONGITUDE, GEOLIFE_SAMPLE
+from skmob.core.trajectorydataframe import TrajDataFrame
+from skmob.core.flowdataframe import FlowDataFrame
+from skmob.preprocessing import detection, clustering
 import shapely
 import folium
 import matplotlib
diff --git a/skmob/measures/tests/test_individual.py b/skmob/measures/tests/test_individual.py
index 548b65a..214b897 100644
--- a/skmob/measures/tests/test_individual.py
+++ b/skmob/measures/tests/test_individual.py
@@ -1,6 +1,6 @@
-from ...core.trajectorydataframe import TrajDataFrame
-from ...utils import constants, gislib
-from ...measures import individual
+from skmob.core.trajectorydataframe import TrajDataFrame
+from skmob.utils import constants, gislib
+from skmob.measures import individual
 
 import numpy as np
 import pandas as pd
diff --git a/skmob/models/tests/test_epr.py b/skmob/models/tests/test_epr.py
index 9e353ab..0b8c255 100644
--- a/skmob/models/tests/test_epr.py
+++ b/skmob/models/tests/test_epr.py
@@ -1,4 +1,4 @@
-from ...utils import constants
+from skmob.utils import constants
 import pandas as pd
 import geopandas as gpd
 import numpy as np
@@ -6,11 +6,11 @@ import shapely
 import pytest
 from contextlib import ExitStack
 
-from ...core.trajectorydataframe import TrajDataFrame
-from ...models.gravity import Gravity
-from ...models.epr import EPR, DensityEPR, SpatialEPR, Ditras
-from ...models.markov_diary_generator import MarkovDiaryGenerator
-from ...preprocessing import detection, clustering
+from skmob.core.trajectorydataframe import TrajDataFrame
+from skmob.models.gravity import Gravity
+from skmob.models.epr import EPR, DensityEPR, SpatialEPR, Ditras
+from skmob.models.markov_diary_generator import MarkovDiaryGenerator
+from skmob.preprocessing import detection, clustering
 
 atol = 1e-12
 
diff --git a/skmob/models/tests/test_geosim.py b/skmob/models/tests/test_geosim.py
index ac05adb..bb65507 100644
--- a/skmob/models/tests/test_geosim.py
+++ b/skmob/models/tests/test_geosim.py
@@ -1,4 +1,4 @@
-from ...utils import constants
+from skmob.utils import constants
 import pandas as pd
 import geopandas as gpd
 import numpy as np
@@ -6,8 +6,8 @@ import shapely
 import pytest
 from contextlib import ExitStack
 from sklearn.metrics import mean_absolute_error
-from ...models.geosim import GeoSim
-from ...core.trajectorydataframe import TrajDataFrame
+from skmob.models.geosim import GeoSim
+from skmob.core.trajectorydataframe import TrajDataFrame
 
 def global_variables():
     # tessellation
diff --git a/skmob/models/tests/test_gravity.py b/skmob/models/tests/test_gravity.py
index 90ac92e..a3cec9c 100644
--- a/skmob/models/tests/test_gravity.py
+++ b/skmob/models/tests/test_gravity.py
@@ -4,9 +4,9 @@ import geopandas as gpd
 import shapely
 import pytest
 import functools
-from ...core.trajectorydataframe import FlowDataFrame
-from ...utils import constants, gislib
-from ...models.gravity import Gravity, exponential_deterrence_func, powerlaw_deterrence_func
+from skmob.core.trajectorydataframe import FlowDataFrame
+from skmob.utils import constants, gislib
+from skmob.models.gravity import Gravity, exponential_deterrence_func, powerlaw_deterrence_func
 
 distfunc = gislib.getDistance
 
diff --git a/skmob/models/tests/test_sts_epr.py b/skmob/models/tests/test_sts_epr.py
index d6abf7b..4a4e8ed 100644
--- a/skmob/models/tests/test_sts_epr.py
+++ b/skmob/models/tests/test_sts_epr.py
@@ -1,4 +1,4 @@
-from ...utils import constants
+from skmob.utils import constants
 import pandas as pd
 import geopandas as gpd
 import numpy as np
@@ -6,10 +6,10 @@ import shapely
 import pytest
 from contextlib import ExitStack
 from sklearn.metrics import mean_absolute_error
-from ...preprocessing import detection, clustering
-from ...models.sts_epr import STS_epr
-from ...core.trajectorydataframe import TrajDataFrame
-from ...models.markov_diary_generator import MarkovDiaryGenerator
+from skmob.preprocessing import detection, clustering
+from skmob.models.sts_epr import STS_epr
+from skmob.core.trajectorydataframe import TrajDataFrame
+from skmob.models.markov_diary_generator import MarkovDiaryGenerator
 
 
 
diff --git a/skmob/utils/tests/test_plots.py b/skmob/utils/tests/test_plots.py
index 077eea6..307aba5 100644
--- a/skmob/utils/tests/test_plots.py
+++ b/skmob/utils/tests/test_plots.py
@@ -3,10 +3,10 @@ import pandas as pd
 import geopandas as gpd
 import shapely
 import pytest
-from ...core.trajectorydataframe import TrajDataFrame, FlowDataFrame
+from skmob.core.trajectorydataframe import TrajDataFrame, FlowDataFrame
 from .. import plot
-from ...utils import constants
-from ...preprocessing import detection, clustering
+from skmob.utils import constants
+from skmob.preprocessing import detection, clustering
 import folium
 import matplotlib
 
