diff --git a/skmob/core/tests/test_trajectorydataframe.py b/skmob/core/tests/test_trajectorydataframe.py
index 0cbd92b..1c43613 100644
--- a/skmob/core/tests/test_trajectorydataframe.py
+++ b/skmob/core/tests/test_trajectorydataframe.py
@@ -141,7 +141,7 @@ class TestTrajectoryDataFrame:
 
     def test_plot_diary(self):
         ax = self.cstdf.plot_diary(self.tdf0[UID].iloc[0])
-        assert isinstance(ax, matplotlib.axes._subplots.Subplot)
+        assert isinstance(ax, matplotlib.axes.Subplot)
 
     @pytest.mark.parametrize('self_loops', [True, False])
     def test_to_flowdataframe(self, self_loops):
